kover-gradle-plugin/build.gradle.kts

```diff
@@ -199,3 +215,18 @@ gradlePlugin {
         }
     }
 }
+
+gradlePlugin {
+    website.set("https://github.com/Kotlin/kotlinx-kover")
+    vcsUrl.set("https://github.com/Kotlin/kotlinx-kover.git")
+
+    plugins {
+        create("KoverSettingsPlugin") {
+            id = "org.jetbrains.kotlinx.kover.settings"
+            implementationClass = "kotlinx.kover.gradle.aggregation.settings.KoverSettingsGradlePlugin"
+            displayName = "Gradle Plugin for Kotlin Code Coverage Tools"
```

Comment:
It has the same `displayName` and `description` as a regular plugin. Won't it be confusing?

Reply:
I changed it a little